package.json

```diff
@@ -69,10 +69,16 @@
   },
   "dependencies": {
     "choo": "6.6.0",
+    "doc-sniff": "^1.0.1",
```

Comment:
I've found two ways to avoid the chore of removing `^` from `package.json`
- while doing `npm install` of new dependency, one can do it with `--save --save-exact`
- to make it default behaviour: `npm config set save-exact true`